src/Network/HTTP/Affjax.purs

```diff
@@ -161,7 +161,7 @@ foreign import unsafeAjax
         }
       }
       xhr.onerror = function (err) {
-        errback(err)();
+        errback(new Error("AJAX request failed"))();
```

Comment:
Would be helpful to include URL, method, etc., in the error message to make diagnosing such troubles easier.